Registry/Registry.EM_COMMON

```diff
@@ -171,6 +171,10 @@ state    real   utrop          ij       dyn_em      1        X     i1  "UTROP"
 state    real   vmaxw          ij       dyn_em      1        Y     i1  "VMAXW"      "V-component of the max wind speed"  "m s-1"
 state    real   vtrop          ij       dyn_em      1        Y     i1  "VTROP"      "V-component of the tropopause wind" "m s-1"
 state    real   erod           ij.      misc        1        -     i012rd  "EROD" "fraction of erodible surface in each grid cell (0-1)"       "none"
+# PSH - begin
```

Comment:
we can remove these identifier lines from the Registry maybe use generic 
for shallow water roughness option